docs/source/whats-new.md

```diff
@@ -4,6 +4,47 @@ What's new
 See [GitHub releases page](https://github.com/OSOceanAcoustics/echopype/releases) for the complete history.
 
 
+# v0.8.2 (2023 November 20)
+
+## Overview
+
+This release includes a number of important performance enhancements, bug fixes, and under-the-hood refactoring to improve code readability.
+
+## Enhancements
+- Full refactor of the functionality to directly storing parsed data to zarr before set_groups_* (#1185, #1218 [NOT MERGED], #1070)
+  - This avoids memory expansion during NaN padding across channel
+- Improve `compute_MVBS` using flox (#1124)
+- Revive and improve `compute_NASC` using flox (#1167)
+- Refactor AZFP XML parser
+  - Parse more parameters and store in the resulting `EchoData` objects (#1135)
+  - Add support for AZFP multiple phase settings (#1182)
+- AZFP pressure data handling
+  - Parse and store AZFP pressure data (#1189)
+  - Add `depth_from_pressure` method and store calculated depth data in Platform group (#1207, #1202 [NOT MERGED])
+- Remove unused mechanism to selectively parse some EK datagrams (#1214)
```

Comment:
If we merge my PR #1226 (look it over when you have a chance), let's add a  new item above this one, with the scope of that PR: In AZFP Environment group, adding two missing variables (though likely filled with nan) and making the addition of temperature and pressure variables contingent on the presence of actual, valid data.

Reply:
I am not able to review those before Friday, so let's defer those to the next release